fenix/app/src/main/java/org/mozilla/fenix/onboarding/WidgetPinnedReceiver.kt

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import android.app.PendingIntent
+import android.appwidget.AppWidgetManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import mozilla.components.support.utils.PendingIntentUtils
+import org.mozilla.fenix.onboarding.view.JunoOnboardingScreen
+
+/**
+ * Receiver required to catch callback from Launcher when prompted
+ * to add search widget from the Juno Onboarding
```

Comment:
```suggestion
 * Receiver required to catch callback from Launcher when prompted
 * to add search widget from the Juno Onboarding.
```
Could we go through the comments in this file and just turn them all into proper sentences by adding the right punctuations?